assets/app/scripts/controllers/overview.js

```diff
@@ -12,9 +12,7 @@ angular.module('openshiftConsole')
     $scope.pods = {};
```

Comment:
This set of scope vars is a little bewildering... could we order/name/doc them in a way that makes it easier to understand the structure of what is stored in these without having to read all the helper functions that build them?